core/config.py

```diff
@@ -178,4 +177,15 @@
 
     # Will be set to viur.core.version.__version__ in viur.core.__init__
     "viur.version": None,
+
+    # Will be set in viur.core.__init__
+    "viur.instance.is_dev_server": False,
```

Comment:
Why shouldn't we set the value directly here?

```suggestion
    "viur.instance.is_dev_server": os.getenv('GAE_ENV') == "localdev",  
```
(import `os` before)